.github/workflows/smart-integ-inferno-test-periodically.yaml

```diff
@@ -67,7 +68,7 @@ jobs:
   custom-integration-tests:
     needs: inferno-test
     name: Run custom integration tests - enableMultiTenancy=${{ matrix.enableMultiTenancy }}
-    environment: FWoA Integ Test Env
```

Comment:
Would this affect the execution of the workflow? Does the workflow depends on FWoA Integ Test Env for anything?